page.go

```diff
@@ -57,14 +60,14 @@ func buildStatusPage() (updates []StatusUpdate, pinnedUpdates []StatusUpdate, er
 		}
 	}
 
-	if len(pinnedUpdates) == 0 {
-		pinnedUpdates = append(pinnedUpdates, StatusUpdate{
-			Text:       config.NominalMessage,
-			SentBy:     config.NominalSentBy,
-			TimeStamp:  "Now",
-			Background: config.StatusOKColor,
-		})
-	}
+	// if len(pinnedUpdates) == 0 {
+	// 	pinnedUpdates = append(pinnedUpdates, StatusUpdate{
+	// 		Text:       config.NominalMessage,
+	// 		SentBy:     config.NominalSentBy,
+	// 		TimeStamp:  "Now",
+	// 		BackgroundClass: config.StatusOKColor,
+	// 	})
+	// }
```

Comment:
If this is no longer needed, just delete this code block.

Reply:
Deleted